packages/shared/src/components/FeedLayout.tsx

```diff
@@ -1,73 +1,80 @@
-import React, { ReactElement, ReactNode, useContext } from 'react';
+import React, { ReactElement, ReactNode, useContext, useMemo } from 'react';
 import { desktop, laptop, laptopL, laptopXL, tablet } from '../styles/media';
-import FeedContext, {
-  defaultFeedContextData,
-  FeedContextData,
-} from '../contexts/FeedContext';
+import FeedContext, { defaultFeedContextData } from '../contexts/FeedContext';
 import { useMedia } from '../hooks';
 import SettingsContext from '../contexts/SettingsContext';
 import useSidebarRendered from '../hooks/useSidebarRendered';
+import { feature } from '../lib/featureManagement';
+import { useFeature } from './GrowthBookProvider';
 
 export type FeedLayoutProps = { children?: ReactNode };
 
-type DetermineFeedSettingsProps = {
-  sidebarExpanded: boolean;
-  sidebarRendered: boolean;
-};
+type FeedSettingsKeys =
+  | 'default'
+  | 'tablet'
+  | 'laptop'
+  | 'laptopL'
+  | 'laptopXL'
+  | 'desktop';
 
-export const feedBreakpoints = [tablet, laptop, laptopL, laptopXL, desktop];
+export type FeedSettings = {
+  pageSize: number;
+  breakpoint?: string;
```

Comment:
Why do we need the breakpoint in the settings?

Reply:
We don't really, just though it's cleaner to have it defined together with the rest, then a separate array like before